content/c-sharp/concepts/math-functions/terms/max/max.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'Math.Max()'
+Description: 'Returns the larger of two specified numbers.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`Math.Max()`** method returns the larger of two specified numbers.
+
+## Syntax
+
+```pseudo
+//returns the greater value from the two numbers
```

Comment:
Rather than a comment, it would be prefreable to have this as a sentence after the code block. And instead of referring to "the two numbers" say "`Math.Max()` returns the greater value from `num1` and `num2`"